### ISSUE TITLE
How to assign units?

### ISSUE THREAD

**Issue by dorahermes** [hed-standard/hed-schemas#145]
Another question, I want to add an example of some quantity (e.g. weight/amplitude) being less than a certain value, but can't find how to add the units, how can I add kilogram to: `(Weight, (Less-than, (Weight/30)))`? thanks!

_Originally posted by @dorahermes in https://github.com/hed-standard/hed-schemas/issues/137#issuecomment-1981588564_

**Comment by IanCa** [comment]
Let me know if this doesn't do what you want:

(Weight, (Less-than, (Weight/30 kg)))

This should also work with placeholders e.g.:
(Weight, (Less-than, (Weight/# kg)))

**Comment by dorahermes** [comment]
Thank you! That works and I can now use this successfully `(Finding-amplitude, (Less-than, (Finding-amplitude/30 v)))`

However, there seems to be an issue using Volt, and I get an error using this: `(Finding-amplitude, (Less-than, (Finding-amplitude/30 Volt)))` 

```
Errors for HED string 1:
UNITS_INVALID: Invalid unit - "Finding-amplitude/30 Volt" valid units are "Volt,v"

```

**Comment by IanCa** [comment]
I'm going to need to touch base with Kay and get back to you, but the root of the problem is Volt shouldn't have been capitalized in the schema originally.

**Comment by IanCa** [comment]
https://github.com/hed-standard/hed-python/pull/884

This should make Volt and volt both work.  The v vs V change will presumably be in 8.3.0 or later.

**Comment by VisLab** [comment]
This has been fixed in 8.3.0 prerelease.  NIST says that SI is V and volt.  The cap changes have been made.
However, going forward, once the hedId assignment becomes permanent -- this sort of case change will require deprecation.
